python/pylibraft/pylibraft/cluster/kmeans.pyx

```diff
@@ -84,37 +84,40 @@ def compute_new_centroids(X,
     Examples
     --------
 
-    >>> import cupy as cp
+    .. code-block:: python
```

Comment:
@lowener Any reason why we are adding python code-blocks? AFAIK sphinx does a good job at parsing the example code section of documentation.

Reply:
Currently the code examples on Sphinx are split into many small blocks, I thought that it was unintentional and I didn't know that it was a shift to a different format.
If this small block format has already been decided then I should revert my changes.
Current
![image](https://github.com/rapidsai/raft/assets/9810050/e8a36a4f-7888-473a-a2b6-e7245884e835)
Proposed
![image](https://github.com/rapidsai/raft/assets/9810050/faff4f84-4a65-41d4-a7e8-049cc503cbaf)

Reply:
Got it, I have a simpler solution for it. See my comment below: https://github.com/rapidsai/raft/pull/1717/files#r1284444539